main.go

```diff
@@ -0,0 +1,86 @@
+package main
```

Comment:
Use this pattern for structuring this code - https://www.alexedwards.net/blog/the-fat-service-pattern

Reply:
Actually ignore above one. Use this pattern instead - https://medium.com/twodigits/java-microservice-layer-architecture-3874a9a6c611

Jaha bhi doubts honge you can ask

Reply:
This is Java ka pattern but very very popular since Java is more popular and widely used as compared to GoLang

Reply:
Thike MCV model karna hain